samigo/samigo-api/src/java/org/sakaiproject/tool/assessment/data/ifc/assessment/EvaluationModelIfc.java

```diff
@@ -39,7 +39,8 @@
   public static final Integer TO_DEFAULT_GRADEBOOK = Integer.valueOf(1);
   //public static Integer TO_SELECTED_GRADEBOOK = new Integer(2);  // this is confusing, we are using 2 for 'None' but the name is confusing, 
   public static final Integer NOT_TO_GRADEBOOK = Integer.valueOf(2);		// so now we added this new constant, SAK-7162
-  public static final Integer TO_SELECTED_GRADEBOOK = Integer.valueOf(3);  // not used, but leave it for now 
+  public static final Integer TO_SELECTED_GRADEBOOK = Integer.valueOf(3);  // not used, but leave it for now
```

Comment:
seems an odd pattern, just use the int value and it will be autoboxed to the Object type. Can you change the others while you are there?